sdk/ml/azure-ai-ml/tests/dsl/unittests/test_dsl_group.py

```diff
@@ -327,21 +327,21 @@ def str_param_pipeline(str_param, group: ParamClass):
         # Note: this is the expected behavior.
         assert (
             pipeline_job._to_dict()["inputs"]["str_param"]
-            == "TestDSLPipeline.test_assign_group_invalid.<locals>.ParamClass(str_param='string_by_default')"
+            == "TestDSLGroup.test_assign_group_invalid.<locals>.ParamClass(str_param='string_by_default')"
         )
 
         pipeline_job.inputs.str_param = "test"
         with pytest.raises(Exception) as e:
             pipeline_job.inputs.group = "test"
         assert "'group' is expected to be a parameter group, but got <class 'str'>." in str(e)
 
-    @pytest.mark.skip(reason="Parameter group item .result() is not supported currently.")
-    def test_parameter_group_result(self):
-        @parameter_group
+    @pytest.mark.skip(reason="Input group item .result() is not supported currently.")
```

Comment:
BTW what's the current error if we do so on an input in group?

Reply:
Calling `result()` on group params will return None, e.g.:

```
        @pipeline()
        def pipeline_with_group(group: ParamClass, int_param: int):
            assert group.str_param.result() is None
```